### PR TITLE
Add fuzzing audit report for Collector

### PR DESCRIPTION
**Pull request by AdamKorcz**
The Opentelemetry Collector has undergone a fuzzing audit in collaboration with the CNCF and @reyang @jpkrohling @codeboten 

The CNCF asks that fuzzing audit reports are stored in the respective projects repository. We thought this community repo is a great way to do that.

 We have done the same for all previous CNCF fuzzing audits. Some examples are:

1. Helm (community repo): [Report](https://github.com/helm/community/blob/main/security-audit/FUZZING_AUDIT_2022.pdf)
2. containerd (website repo): [Report](https://github.com/containerd/containerd.io/blob/main/static/img/ADA-fuzzing-audit-21-22.pdf)
3. Envoy (core repo): [Report](https://github.com/envoyproxy/envoy/blob/main/docs/security/audit_fuzzer_adalogics_2021.pdf)
4. Lima (core repo): [Report](https://github.com/lima-vm/lima/blob/master/docs/reports/Ada-Logics-Lima-fuzzing-audit-2024.pdf)
5. Crossplane (core repo): [Report](https://github.com/crossplane/crossplane/blob/main/security/ADA-fuzzing-audit-22.pdf)

### CODE REVIEW

**Review by austinlparker — APPROVED** [review]

**Review by mx-psi — APPROVED** [review]

**Comment by reyang** [comment]
Thanks @AdamKorcz!

**Review by reyang — APPROVED** [review]
LGTM.

I suggest that we change "Opentelemetry" to "OpenTelemetry" (check https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification#project-naming) in the pdf file, it'll be better if there is a way for folks to add comments in the PR if this a markdown file, but I understand that CNCF prefers pdf.

image

**Comment by reyang** [comment]
@svrnm I think either this repo or https://github.com/open-telemetry/opentelemetry.io/ would work. Just want to get you informed in case you might have a strong preference.

**Comment by AdamKorcz** [comment]
> LGTM.
> 
> I suggest that we change "Opentelemetry" to "OpenTelemetry" (check https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification#project-naming) in the pdf file, it'll be better if there is a way for folks to add comments in the PR if this is a markdown file, but I understand that CNCF prefers pdf.
> 
> image

Updated in https://github.com/open-telemetry/community/pull/2432/commits/299269103d0ca334ede55cd51a692a923f54cd4e

File: reports/ADA_Logics-collector-fuzzing-audit-2024.pdf
Comment (arminru) [inline reports/ADA_Logics-collector-fuzzing-audit-2024.pdf:1]: `reports/` is a new top-level folder.
@austinlparker @svrnm is this a consistent addition to the repo or should it be included elsewhere in the existing structure?

Should we add a README.md there with a brief description/ToC, and link to it from root?

**Review by arminru — APPROVED** [review]

**Comment by svrnm** [comment]
> @svrnm I think either this repo or [open-telemetry/opentelemetry.io](https://github.com/open-telemetry/opentelemetry.io/) would work. Just want to get you informed in case you might have a strong preference.

No strong preference, both is fine with me. 

What we could do, if we want to have it on the website, is pairing it with a blog post, so it is not just put somewhere, but also shared with our community and end users, similar to https://opentelemetry.io/blog/2024/security-audit-results/

**Comment by reyang** [comment]
> > @svrnm I think either this repo or [open-telemetry/opentelemetry.io](https://github.com/open-telemetry/opentelemetry.io/) would work. Just want to get you informed in case you might have a strong preference.
> 
> No strong preference, both is fine with me.
> 
> What we could do, if we want to have it on the website, is pairing it with a blog post, so it is not just put somewhere, but also shared with our community and end users, similar to https://opentelemetry.io/blog/2024/security-audit-results/

@svrnm what would be the next step? Do you plan to merge this PR?

**Comment by svrnm** [comment]
> > > @svrnm I think either this repo or [open-telemetry/opentelemetry.io](https://github.com/open-telemetry/opentelemetry.io/) would work. Just want to get you informed in case you might have a strong preference.
> > 
> > 
> > No strong preference, both is fine with me.
> > What we could do, if we want to have it on the website, is pairing it with a blog post, so it is not just put somewhere, but also shared with our community and end users, similar to https://opentelemetry.io/blog/2024/security-audit-results/
> 
> @svrnm what would be the next step? Do you plan to merge this PR?

Merging works for me if we have the required amount of approvals, but if someone would sit down and write a few words I am also happy to have a blog post for it on the otel.io blog. Where the pdf lives doesn't matter, we can keep it here or we can have it as part of the blog post on the website.

**Review by svrnm — APPROVED** [review]

File: reports/ADA_Logics-collector-fuzzing-audit-2024.pdf
Comment (trask) [reply reports/ADA_Logics-collector-fuzzing-audit-2024.pdf:1]: I'm going to resolve this discussion so I can merge. I think it's a good follow-up if anyone has concrete ideas, or maybe next time we add a report...

**Pull request merged**